dll/directx/d3d9/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+
+spec2def(d3d9.dll d3d9.spec ADD_IMPORTLIB)
```

Comment:
Do we need `ADD_IMPORTLIB` here?

Reply:
Yes, since d3d9 is linked to dxdiag and dxdiagn: https://github.com/reactos/reactos/blob/c1195f5300b3116594bb27b23fb16e321775f36b/base/applications/dxdiag/CMakeLists.txt#L25
https://github.com/reactos/reactos/blob/e18e7b100c236835ffa6ca06114a5c8551becec5/dll/directx/wine/dxdiagn/CMakeLists.txt#L22